src/wiki/development/build-instructions.md

```diff
@@ -33,17 +33,45 @@ Getting the project to build and run on Linux is easy if you use any modern and
 ### Build dependencies
 
 - A C++ compiler capable of building C++11 code.
-- Qt Development tools 5.12 or newer (`qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools libqt5core5a libqt5network5 libqt5gui5` on Debian-based systems)
-  - Alternatively you can also use Qt 6.0 or newer (`qt6-base-dev qtchooser qt6-base-dev-tools libqt6core6 libqt6core5compat6 libqt6network6 libqt6network6` on Debian (testing/unstable) based systems), if you prefer it.
-- cmake 3.15 or newer (`cmake` on Debian-based system)
-- extra-cmake-modules (`extra-cmake-modules` on Debian-based system)
-- zlib (`zlib1g-dev` on Debian-based system)
-- Java JDK (`openjdk-17-jdk` on Debian-based system)
-- GL headers (`libgl1-mesa-dev` on Debian-based system)
-- scdoc if you want to generate manpages (`scdoc` on Debian-based system)
+- Qt Development tools 5.12 or newer
+  - Alternatively you can also use Qt 6.0 or newer
+- cmake 3.15 or newer
+- extra-cmake-modules
+- zlib
+- Java JDK
+- GL headers
+- scdoc if you want to generate manpages
 
 You can use IDEs, like KDevelop, QtCreator or CLion to open the CMake project, if you want to work on the code.
 
+## Distributions
+
+The following distribution instructions are with fresh instances of itself with podman. They are opinionated and use GCC for their C compiler.
+
+### Fedora, CentOS Stream/RHEL
+
+```
+sudo dnf install git gcc-c++ cmake java-latest-openjdk-devel zlib-devel extra-cmake-modules qtchooser
+
+# Qt 5
+sudo dnf install qt5-qtbase-devel
+
+# Qt 6
+sudo dnf install qt6-qtbase-devel
```

Comment:
definetly needs more dependencies for building polymc

Reply:
That's odd -- it builds perfectly fine on podman. I'll try a VM when I get home to see of the build works there.

Reply:
well polymc needs core5compat on qt6, soooo

Reply:
huh, probably ran it wrong then -- I'll be able to check once I get back.